<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInfoBarStatus</class>
 <widget class="QFrame" name="CInfoBarStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Info status bar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../blackmisc/blackmisc.qrc">
    <normaloff>:/own/icons/own/swift/swift24.png</normaloff>:/own/icons/own/swift/swift24.png</iconset>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="hl_InfoBarStatus">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lbl_Network">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Network</string>
     </property>
     <property name="text">
      <string>Net</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="BlackGui::CLed" name="led_Network" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Simulator">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Simulator</string>
     </property>
     <property name="text">
      <string>Sim</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="BlackGui::CLed" name="led_Simulator" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_DBus">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>DBus</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="BlackGui::CLed" name="led_DBus" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Ptt">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Push to talk</string>
     </property>
     <property name="text">
      <string>Ptt</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="BlackGui::CLed" name="led_Ptt" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Audio">
     <property name="toolTip">
      <string>Audio</string>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>Audio</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="BlackGui::CLed" name="led_Audio" native="true"/>
   </item>
   <item>
    <spacer name="hs_InfoStatusBar">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>239</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::CLed</class>
   <extends>QWidget</extends>
   <header>blackgui/led.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../blackmisc/blackmisc.qrc"/>
 </resources>
 <connections/>
</ui>
